CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.3.1] unreleased
```

Comment:
do you need/want to update this to today's date?

Reply:
Yeah I will once its ready for release. I didn't know which date this one would be getting released